<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>881</width>
      <height>489</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutLoadFile">
       <item>
        <widget class="QPushButton" name="loadFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>loadFile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="textLayout">
       <item>
        <widget class="QTextEdit" name="fileText">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="infoLayout">
       <item>
        <layout class="QHBoxLayout" name="spaziEPunteggiaturaInclusa">
         <item>
          <widget class="QLabel" name="caratteriSpaziEPunteggiaturaInclusi">
           <property name="text">
            <string>caratteri (punteggiatura e spazi inclusi):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numeroCarInc">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="spaziEPunteggiaturaEsclusa">
         <item>
          <widget class="QLabel" name="caratteriSpaziEsclusi">
           <property name="text">
            <string>caratteri (spazi esclusi):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numeroCarEsc">
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutNFrasi">
       <item>
        <widget class="QLabel" name="numeroFrasi">
         <property name="text">
          <string>numero frasi:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nFr">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutNParole">
       <item>
        <widget class="QLabel" name="numeroParole">
         <property name="text">
          <string>numero parole:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nPar">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutNparagrafi">
       <item>
        <widget class="QLabel" name="numeroParagrafi">
         <property name="text">
          <string>numero paragrafi:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nPar_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="csvFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>crea un csv file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="graphButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>mostraGrafico</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chiudi">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>chiudi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
